docs/standard/microservices-architecture/docker-application-development-process/docker-app-development-workflow.md

```diff
@@ -240,120 +226,109 @@ services:
 
-**Figure 5-8**. The **docker-compose** tree node added in Visual Studio 2017 Solution Explorer
-
-You could deploy a multi-container application with a single docker-compose.yml file by using the docker-compose up command. However, Visual Studio adds a group of them so you can override values depending on the environment (development versus production) and execution type (release versus debug). This capability will be explained in later sections.
-
-![](./media/image12.png)
+![Step 5 - Run app graphic](./media/image12.png)
 
 ## Step 5. Build and run your Docker application
 
 If your application only has a single container, you can run it by deploying it to your Docker host (VM or physical server). However, if your application contains multiple services, you can deploy it as a composed application, either using a single CLI command (docker-compose up), or with Visual Studio, which will use that command under the covers. Let’s look at the different options.
 
-### Option A: Running a single-container with Docker CLI
+### Option A: Run a single-container app
 
-You can run a Docker container using the docker run command, as in Figure 5-9:
+#### Docker CLI
+
+You can run a Docker container using the docker run command:
 
 ```console
   docker run -t -d -p 80:5000 cesardl/netcore-webapi-microservice-docker:first
 ```
 
-![](./media/image13.png)
-
-**Figure 5-9**. Running a Docker container using the docker run command
+![Running a Docker container using the docker run command](./media/image13.png)
 
 In this case, the command binds the internal port 5000 of the container to port 80 of the host machine. This means that the host is listening on port 80 and forwarding to port 5000 on the container.
 
-### Option B: Running a multi-container application
+#### Visual Studio
 
-In most enterprise scenarios, a Docker application will be composed of multiple services, which means you need to run a multi-container application, as shown in Figure 5-10.
+If you haven't added container orchestrator support, you can also run a single container app in Visual Studio by pressing **F5**. The container runs locally using docker run.
 
-![](./media/image14.png)
+### Option B: Run a multi-container app
 
-**Figure 5-10**. VM with Docker containers deployed
+In most enterprise scenarios, a Docker application will be composed of multiple services, which means you need to run a multi-container application:
 
-#### Running a multi-container application with the Docker CLI
+![Graphic showing VM with Docker containers deployed](./media/image14.png)
 
-To run a multi-container application with the Docker CLI, you can run the docker-compose up command. This command uses the docker-compose.yml file that you have at the solution level to deploy a multi-container application. Figure 5-11 shows the results when running the command from your main project directory, which contains the docker-compose.yml file.
+#### Run a multi-container application with the Docker CLI
```

Comment:
should you do the same change you did for Option A and just say Docker CLI and Visual Studio?